script/Ciel/Day 8/08_01_CIEL08_4.txt

```diff
@@ -26,17 +26,18 @@ My vague monologue, intended for no one in particular, faded away, like sucked u
 [sha:5d9c65f8e6b88bc9430ff7cfe94dddcb367bc07c]{
 -- Page 335, Offset 19095.
 -- 「いや別に。アルクェイドには関係のないコトだ、気にしないでくれ」
-"No, nothing. That doesn't concern you, don't worry about it."
+"No, nothing. It doesn't concern you, so don't worry about it."
 }
 [sha:68fab313acc807bf6f8eefa5764784c96b01050b]{
 -- Page 336, Offset 19096.
 -- それに、約束で言えばアルクェイドとの約束の方が先なのだ。
-Besides, I also made a promise to Arcueid and it takes priority.
+// any chance 先 means chronologically first instead?
+Besides, in terms of promises, the one with Arcueid takes priority.
```

Comment:
I think just 'Besides, my promise to Arcueid takes priority.' is sufficient here.